pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -879,3 +879,50 @@ func (br *OVSBridge) getHardwareOffload() (bool, Error) {
 func (br *OVSBridge) GetOVSDatapathType() OVSDatapathType {
 	return br.datapathType
 }
+
+// SetInterfaceTypeInternal modifies the OVS Interface type as "internal", and return until the ofport is allocated by
+// OVS. This function is used on Windows when the Pod interface is created after the OVS port creation.
+func (br *OVSBridge) SetInterfaceTypeInternal(name string) (int32, Error) {
```

Comment:
Can we make the method more generic, i.e. `SetInterfaceType(name, type string)`?